### DISCUSSION TITLE
Jackson 3.0.0 default for `SerializationFeature.WRITE_DATES_AS_TIMESTAMPS`

### DISCUSSION THREAD

**Discussion by cowtowncoder** [FasterXML/jackson-future-ideas#77] [Polls]
(part of #72 uber-Discussion)

As of Jackson 2.x, `SerializationFeature.WRITE_DATES_AS_TIMESTAMPS` defaults to `true` for backwards-compatibility. A suggestion has been made:

https://github.com/FasterXML/jackson-databind/issues/4845

to change it to `false` instead, to improve JSON readability by defaulting to textual serialization over numeric timestamps which, while more efficient, are not human readable.
We cannot make the change in Jackson 2.x, but could change it for 3.0.0.

So let's see if there is consensus on whether to make proposed change.

**Comment by JooHyukKim** [comment]
Seems like we can go push through `false` as default?

**Reply by cowtowncoder** [reply]
Yes.

**Comment by cowtowncoder** [comment]
After almost 2 months, clear results: proposal approved.
